doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1272 @@
+  * New arguments to the :ref:`ug_matter_gs_tools_matter_west_commands_zap_tool_gui` to provide a custom cache directory and add new clusters to Matter Data Model.
+  * :ref:`ug_matter_debug_snippet`.
+  * Storing Matter key materials in the :ref:`matter_platforms_security_kmu`.
+  * A new section :ref:`ug_matter_device_low_power_calibration_period` in the :ref:`ug_matter_device_low_power_configuration` page.
+  * A new section :ref:`ug_matter_gs_tools_opp` in the :ref:`ug_matter_gs_tools` page.
+
+* Updated:
+
+  * By disabling the :ref:`mpsl` before performing factory reset to speed up the process.
+  * The :ref:`ug_matter_device_low_power_configuration` page to mention the `nWP049 - Matter over Thread: Power consumption and battery life`_ and `Online Power Profiler for Matter over Thread`_ as useful resources in optimizing the power consumption of a Matter device.
+
+Matter fork
++++++++++++
+
+* Added a new ``kFactoryReset`` event that is posted during factory reset.
+  The application can register a handler and perform additional cleanup.
+
+Thread
+------
+
+* Added:
+
+  * Support for storing the Thread key materials in the :ref:`ug_ot_thread_security_kmu`.
+  * The :ref:`ug_ot_thread_security` user guide describing the security features of the |NCS| OpenThread implementation.
+
+Zigbee
+------
+
+* Removed all Zigbee resources.
+  They are now available as separate `Zigbee R22`_ and `Zigbee R23`_ add-on repositories.
+
+Wi-Fi®
+------
+
+* The :ref:`ug_wifi_regulatory_certification` documentation is now moved under :ref:`ug_wifi` protocol page.
+
+Applications
+============
+
+* Removed the Asset Tracker v2 application.
+  For development of asset tracking applications, refer to the `Asset Tracker Template <Asset Tracker Template_>`_.
+
+  The factory-programmed Asset Tracker v2 firmware is still available to program the nRF91 Series devices using the `Programmer app`_, `Quick Start app`_ , and the `Cellular Monitor app`_.
+
+Machine learning
+----------------
+
+* Updated:
+
+  * The application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+  * The Edge Impulse URI configuration to use the new model location.
+
+IPC radio firmware
+------------------
+
+* Updated:
+
+  * The application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system in all devices that contain MRAM, such as the nRF54H Series devices.
+  * The documentation of applications and samples that use the IPC radio firmware as a :ref:`companion component <companion_components>` to mention its usage when built with :ref:`configuration_system_overview_sysbuild`.
+
+Matter bridge
+-------------
+
+* Updated by enabling Link Time Optimization (LTO) by default for the ``release`` configuration.
+* Removed support for the nRF54H20 devices.
+
+nRF5340 Audio
```

Comment:
The PR https://github.com/nrfconnect/sdk-nrf/pull/21285 should probably be mentioned here, somehow. But I don't know if this is a highlight or an `Added` or how to word it correctly

Reply:
Suggest: "Added more information on new DNs and QDIDs"

Also with a link to the QDID/DN table for 5340.

Reply:
Added.